src/status_im/ui/screens/views.cljs

```diff
@@ -1,8 +1,8 @@
 (ns status-im.ui.screens.views
   (:require-macros [status-im.utils.views :refer [defview letsubs]])
   (:require [re-frame.core :refer [dispatch]]
-            [status-im.utils.platform :refer [android?]]
-            [status-im.ui.components.react :refer [view modal]]
+            [status-im.utils.platform :refer [android? ios?]]
+            [status-im.ui.components.react :refer [view safe-area-view modal dimensions]]
```

Comment:
Can we turn these into react/foo etc? Generally preferable (think these are leftovers)